docker-images/jmxtrans/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM jmxtrans/jmxtrans
```

Comment:
I wonder if we should build our own image from CentOS base as we do it everywhere else. That will allow us to have the same OS and tooling inside but also to respin the images when some important CVEs hit etc. Like this it is a lot more out of our control.

We should also use some tag here if nothing else (probably not latest).